src/Illuminate/Session/Store.php

```diff
@@ -21,6 +21,13 @@ class Store implements Session
 {
     use Macroable;
 
+    /**
+     * The length used when generating session IDs.
+     *
+     * @var int
+     */
+    private const SESSION_ID_LENGTH = 40;
```

Comment:
I'd change its visibility to `protected`, so subclasses can override this value if needed.

Also, on the `laravel/framework` codebase:

- There are no `private const`
- There is a single `protected const` on (`Illuminate\Support\Js`)
- All other class constants have no modifier, which makes them `public` by default.